javaScript/index.js

```diff
@@ -237,10 +242,19 @@ const validateInputs = () => {
   } else {
     setSuccess();
     mainForm.submit();
+    clientStore.storeEmail = localStorage.setItem('emailKey', clientStore.storeEmail);
+    clientStore.storeName = localStorage.setItem('nameKey', clientStore.storeName);
   }
 };
 
 mainForm.addEventListener('submit', (e) => {
   e.preventDefault();
   validateInputs();
+});
+
+document.addEventListener('DOMContentLoaded', () => {
+  if (localStorage.getItem('emailKey') && localStorage.getItem('nameKey')) {
```

Comment:
- [ ] After filling out the form, and not submitting it, the entered values should be saved in the browser but currently they are not saved I suggest that you use `JSON.stringify` because it helps to save to the local storage data object  turned into a string

Reply:
- [ ] Please make sure that the contact form information is persistent to page reloads